src/DynamicNLPModels.jl

```diff
@@ -29,11 +33,130 @@ function LQDynData(
     ul = (similar(x0,nu) .= -Inf),
     uu = (similar(x0,nu) .=  Inf)
     )
-    # TODO
+
 end
 
 function QPData(dnlp::LQDynData{VT,MT}) where {VT,MT}
-    # TODO
+    
+end
+
+""" 
+Build the (sparse) H matrix for quadratic models from Q and R matrices 
```

Comment:
Checkout the guidelines for docstring https://docs.julialang.org/en/v1/manual/documentation/

The docstring should include what are the arguments for the function. E.g., `?push!` gives me:

```
  push!(collection, items...) -> collection


  Insert one or more items in collection. If collection is an ordered
  container, the items are inserted at the end (in the given order).

  Examples
  ≡≡≡≡≡≡≡≡≡≡

  julia> push!([1, 2, 3], 4, 5, 6)
  6-element Vector{Int64}:
   1
   2
   3
   4
   5
   6


  If collection is ordered, use append! to add all the elements of another
  collection to it. The result of the preceding example is equivalent to
  append!([1, 2, 3], [4, 5, 6]). For AbstractSet objects, union! can be used
  instead.

  See sizehint! for notes about the performance model.

  See also pushfirst!.
```